common/src/test/java/io/seata/common/util/ReflectionUtilTest.java

```diff
@@ -0,0 +1,75 @@
+package io.seata.common.util;
+
+import java.io.Serializable;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class ReflectionUtilTest {
+
+    @Test
+    public void testGetClassByName() throws Exception {
+        Assertions.assertEquals(String.class,
+                ReflectionUtil.getClassByName("java.lang.String"));
+    }
+
+    @Test
+    public void testGetFieldValue() throws Exception {
```

Comment:
Maybe replace the general `throws Exception` with the specific exception thrown here (similarly for other tests)

Reply:
Done